apps/protailwind/src/pages/unsubscribed.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+import Layout from 'components/layout'
+import {useRouter} from 'next/router'
+import {Signature} from './confirm'
+
+const Unsubscribed = () => {
+  const router = useRouter()
+  const sequenceId = Number(router.query?.sequence)
+
+  const message = (sequenceId: number) => {
+    switch (sequenceId) {
+      case 1: // unsubscribed: email course
+        return 'Pro Tailwind email course'
+      case 2: // unsubscribed: live workshop info
+        return 'Pro Tailwind live workshops list'
```

Comment:
suggestion: send "slugs" versus ints

Reply:
I agree with this suggestion, otherwise looks good to me! @garrettdimon 👍